docs/development/getting_started.rst

```diff
@@ -1,163 +1,101 @@
-candidate version is not at least 1.8.5, you should download the current version
-from the `Vagrant Downloads page`_ and then install it.
+Writers
+~~~~~~~
 
-.. code:: sh
+Technical writers are invited to review `this documentation
+<https://docs.securedrop.org/>`_ and fix any mistake in accordance to
+the :doc:`documentation guidelines <documentation_guidelines>`.
 
-    # If your OS vagrant is recent enough
-    sudo apt-get install vagrant
-    # OR this, if you downloaded the deb package.
-    sudo dpkg -i vagrant.deb
+Documentation tasks (bug and improvements), ordered by priority:
 
-We recommend using the stable version of Docker CE (Community Edition) which can
-be installed via the official documentation links:
+* `High <https://github.com/freedomofpress/securedrop/issues?q=is%3Aopen+is%3Aissue+label%3Adocs+label%3AP-high>`_
+* `Unspecified <https://github.com/freedomofpress/securedrop/issues?q=is%3Aopen+is%3Aissue+label%3Adocs>`_
+* `Low <https://github.com/freedomofpress/securedrop/issues?q=is%3Aopen+is%3Aissue+label%3Adocs+label%3AP-low>`_
 
-* `Docker CE for Ubuntu`_
-* `Docker CE for Debian`_
+Devops
+~~~~~~
 
-.. _`Vagrant Downloads page`: https://www.vagrantup.com/downloads.html
-.. _`GitHub #932`: https://github.com/freedomofpress/securedrop/pull/932
-.. _`GitHub #1381`: https://github.com/freedomofpress/securedrop/issues/1381
-.. _`Docker CE for Ubuntu`: https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/
-.. _`Docker CE for Debian`: https://docs.docker.com/engine/installation/linux/docker-ce/debian/
+TBD
 
-.. warning:: We do not recommend installing vagrant-cachier. It destroys apt’s
-            state unless the VMs are always shut down/rebooted with Vagrant,
-            which conflicts with the tasks in the Ansible playbooks. The
-            instructions in Vagrantfile that would enable vagrant-cachier are
-            currently commented out.
+Release managers
+~~~~~~~~~~~~~~~~
 
-.. todo:: This warning is here because a common refrain during hackathons for
-          SecureDrop a while back was "setting up VMs is too slow, you should
-          use vagrant-cachier". We tried it and it had some nasty interactions
-          with Ansible, so we dropped it, and added this note to prevent other
-          people from making the same suggestion. Eventually, we should: (i)
-          Build our own base boxes to dramatically cut down on provisioning
-          times (ii) Remove this note as well as the commented vagrant-cachier
-          lines from the Vagrantfile
+TBD
 
-VirtualBox should be at least version 5.x. See `GitHub #1381`_ for documentation
-of incompatibility with the older VirtualBox 4.x release series.
+Translators
+~~~~~~~~~~~
 
-Finally, install Ansible so it can be used with Vagrant to automatically
-provision VMs. We recommend installing Ansible from PyPi with ``pip`` to ensure
-you have the latest stable version.
+If English is not your mother tongue, your are kindly invited to help
```

Comment:
I think "native tongue" would be better (more gender neutral).

Reply:
done